uaclient/status.py

```diff
@@ -707,6 +725,7 @@ def format_tabular(status: Dict[str, Any]) -> str:
 
     if status["origin"] != "free":
         pairs.append(("Valid until", format_expires(status["expires"])))
+        tech_support_level = status["contract"]["tech_support_level"]
```

Comment:
well I just copypastaed this here from above - I believe those filelds are mandatory when attached, we were accessing those directly already. If we want to `.get` the status dict data, there are more entries being directly accessed, and I think it would be better to address that separately/

Reply:
I agree we are doing that on other parts of the code (which we also need to change), but since we are touching that part already, I think we should already make the change. 

And I agree those fields are mandatory, but thinks can change and weird scenarios can happen, so I think we should be defensive here just in case